OWNERS.md

```diff
@@ -29,6 +30,10 @@ See [CODEOWNERS](CODEOWNERS) for automatic PR assignment.
 * Muvaffak Onus <monus@upbound.io> ([muvaf](https://github.com/muvaf))
 * Hasan Turken <hasan@upbound.io> ([turkenh](https://github.com/turkenh))
 
+## Reviewers
+
+* Max Blatt ([MisterMX](https://github.com/MisterMX))
```

Comment:
@MisterMX Let me know if you prefer your full name here, and whether you'd like me to include an email address.

Reply:
Yes you can use my full name. It is `Maximilian Blatt` and my email is `maximilian.blatt@accenture.com`.